code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,293 @@
+
+/obj/item/slimecross/warping/sepia
+	colour = "sepia"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that holds any living creature in its influence under stasis."
+
+/obj/item/slimecross/warping/cerulean
+	colour = "cerulean"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that displays a hologram of the last creature to cross it."
+
+/obj/item/slimecross/warping/pyrite
+	colour = "pyrite"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that paints things that cross its area a rainbow of colors."
+
+/obj/item/slimecross/warping/red
+	colour = "red"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that increases the effectiveness of attacks made from within it."
+
+/obj/item/slimecross/warping/green
+	colour = "green"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that converts sheets of plasma to resin, allowing you to form xenomorphic structures."
+
+/obj/item/slimecross/warping/pink
+	colour = "pink"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that makes hugging anyone standing in it a pleasant experience."
+
+/obj/item/slimecross/warping/gold
+	colour = "gold"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that accepts coinage for items drawn from an unknown dimension."
+
+/obj/item/slimecross/warping/oil
+	colour = "oil"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that absorbs explosions, converting them into a gooey energy."
+
+/obj/item/slimecross/warping/black
+	colour = "black"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that summons the spirits of the imminently deceased."
+
+/obj/item/slimecross/warping/lightpink
+	colour = "light pink"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that stops simple creatures from crossing its boundary."
+
+/obj/item/slimecross/warping/adamantine
+	colour = "adamantine"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that crystallizes materials based on nearby ore nodes."
+
+/obj/item/slimecross/warping/rainbow
+	colour = "rainbow"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that opens a gate to a pocket dimension."
+
+///////////////////////////////////////////////
+//////////////////SLIME RUNES//////////////////
+///////////////////////////////////////////////
+
+/obj/effect/slimerune/grey
+	name = "grey rune"
+	desc = "The center has an outline of a slime extract in it."
+
+/obj/effect/slimerune/grey/examine(mob/user)
+	..()
+	var/obj/item/slimecross/warping/grey/ex = extract
+	to_chat(user, "<span class='notice'>It has absorbed [ex.num_absorbed] extract[(ex.num_absorbed != 1) ? "s":""].</span>")
+	to_chat(user, "<span class='notice'>At 8 extracts, it can be manually fed an extract to produce a slime of that color.</span>")
```

Comment:
Why do you need EIGHT extracts to make a new slime? Why not, like, two? Or even just a 1:1 slme extract:slime ratio?

Reply:
:shrug:
Why not, I guess. Two slimes in a max upgrade extractor to get one.